# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/wasm/jsapi/function/constructor.tentative.any.js.ini
# Commit: 3b62bd57747f
# Full Hash: 3b62bd57747f21c800a93da325d999fa5efb3850
# Author: Jessica Tallon <jtallon@igalia.com>
# Date: 2021-09-03 03:51:58
# Regressor Bug: 1715275
# File Overlap Count: 1
# Description:
#   Bug 1715275 - Add support for a JS WebAssembly.Function object r=rhunt
#   
#   This introduces a new Constructor WebAssembly.Function which is part of
#   the WASM JS-API type reflection work. This allows normal JS functions to
#   be wrapped with a type and used with WebAssembly objects (e.g. Tables).
# ==============================================================================

diff -r f6f4514d09bf -r 3b62bd57747f testing/web-platform/meta/wasm/jsapi/function/constructor.tentative.any.js.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/wasm/jsapi/function/constructor.tentative.any.js.ini	Thu Sep 02 15:58:18 2021 +0000
@@ -0,0 +1,57 @@
+[constructor.tentative.any.worker.html]
+    [construct with JS function]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing parameters]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-string parameters & results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-existant parameter and result type]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-function object]
+        expected:
+            if release_or_beta: FAIL
+[constructor.tentative.any.html]
+    [construct with JS function]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing parameters]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-string parameters & results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-existant parameter and result type]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-function object]
+        expected:
+            if release_or_beta: FAIL
+[constructor.tentative.any.js]
+    [construct with JS function]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing parameters]
+        expected:
+            if release_or_beta: FAIL
+    [fail with missing results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-string parameters & results]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-existant parameter and result type]
+        expected:
+            if release_or_beta: FAIL
+    [fail with non-function object]
+        expected:
+            if release_or_beta: FAIL